lodash as a module dependency (instead of devDependency)

I've writen a script that requires `@hoodie/client`, just as described in the readme, but it throwed an error on require:

```
>> Error: Cannot find module 'lodash/defaultsDeep'
```

I suppose that's because my app's dependency tree ends up including an older version on lodash, and once `@hoodie/client` does not specify lodash as a dependency (only as a devDependency) it ends up using that older version, that does not have that file...

I tried to create a `node_modules` dir inside `node_mocules/@hoodie/client` and copy a newer version of lodash there and the error has gone.

Also, looking at the places where `defaultsDeep` is used in this module, it seems to me it's actually not only a dev dependency..

--- a/package.json
+++ b/package.json
@@ -12,6 +12,7 @@
     "@hoodie/store": "^1.0.0",
     "humble-localstorage": "^1.4.2",
     "lie": "^3.0.1",
+    "lodash": "^4.11.1",
     "nets": "^3.2.0"
   },
   "devDependencies": {
@@ -19,7 +20,6 @@
     "greenkeeper-postpublish": "^1.0.0",
     "istanbul": "^0.4.1",
     "istanbul-coveralls": "^1.0.3",
-    "lodash": "^4.11.1",
     "mkdirp": "^0.5.1",
     "npm-run-all": "^3.0.0",
     "proxyquire": "^1.7.6",

